mlir/include/mlir/Query/Matcher/SliceMatchers.h

```diff
@@ -112,7 +112,8 @@ bool BackwardSliceMatcher<Matcher>::matches(
     }
     return true;
   };
-  getBackwardSlice(rootOp, &backwardSlice, options);
+  auto result = getBackwardSlice(rootOp, &backwardSlice, options);
```

Comment:
```suggestion
  LogicalResult result = getBackwardSlice(rootOp, &backwardSlice, options);
```

expand `auto` unless the type is obvious from statement-level context or obnoxious to spell out.